docs_app/content/guide/installation.md

```diff
@@ -22,7 +22,7 @@ To import only what you need using pipeable operators:
 import { of } from 'rxjs';
 import { map } from 'rxjs/operators';
 
-of(1,2,3).pipe(map(x => x + '!!!')); // etc
+of(1,2,3).pipe(map(x => x + '!!!')); 
```

Comment:
Hi there. There is an empty space left after `;` which might be removed. The other one couple of lines below is removed.

Reply:
thanks for spotting this